homework/04LessonFour.js

```diff
@@ -72,7 +86,7 @@ var nestedObjects = {
       hometown: 'Omaha'
     },
     'bob roberts': {
-      starring: 'John Cusack'
+      starring: 'Tim Robbins'
```

Comment:
Again, use bracket and/or dot notation here to access and assign values to these objects.